src/BeeNet/BeeNodeClient.cs

```diff
@@ -20,21 +22,30 @@ public class BeeNodeClient : IBeeNodeClient, IDisposable
         public BeeNodeClient(
             string baseUrl = "http://localhost/",
             int? gatewayApiPort = 1633,
-            int? debugApiPort = 1635)
+            int? debugApiPort = 1635,
+            ClientVersions? version = ClientVersions.v1_4)
```

Comment:
Ok having version with default value, but don't consider it nullable. Also following assignment `version ??= ClientVersions.v1_4;` is not required.